R/scale-.R

```diff
@@ -607,6 +607,12 @@ check_breaks_labels <- function(breaks, labels, call = NULL) {
   if (is.null(breaks)) {
     return(TRUE)
   }
+  if (identical(breaks, NA)) {
+    cli::cli_abort(
+      "Invalid {.arg breaks} specification. Use {.code NULL}, not {.code NA}.",
+      call = call
+    )
+  }
```

Comment:
This is an adjacent change that seemed like a good idea because we can now omit this check in the `get_breaks()` calculation. It is the reason why the unit tests had to be adapted.